src/main/java/build/buildfarm/server/BuildFarmServer.java

```diff
@@ -107,40 +114,67 @@ public BuildFarmServer(
-            .addService(healthStatusManager.getHealthService())
-            .addService(new ActionCacheService(instance, config.getAcPolicy()))
-            .addService(new CapabilitiesService(instance))
-            .addService(
-                new ContentAddressableStorageService(
-                    instance,
-                    /* deadlineAfter=*/ config.getCasWriteTimeout().getSeconds(),
-                    TimeUnit.SECONDS
-                    /* requestLogLevel=*/ ))
-            .addService(
-                new ByteStreamService(
-                    instance,
-                    /* writeDeadlineAfter=*/ config.getBytestreamTimeout().getSeconds(),
-                    TimeUnit.SECONDS))
-            .addService(
-                new ExecutionService(
-                    instance,
-                    config.getExecuteKeepaliveAfterSeconds(),
-                    TimeUnit.SECONDS,
-                    keepaliveScheduler,
-                    getMetricsPublisher(config.getMetricsConfig())))
-            .addService(new OperationQueueService(instance))
-            .addService(new OperationsService(instance))
-            .addService(new AdminService(config.getAdminConfig(), instance))
-            .addService(new FetchService(instance))
-            .addService(ProtoReflectionService.newInstance())
-            .addService(new PublishBuildEventService(config.getBuildEventConfig()))
-            .intercept(TransmitStatusRuntimeExceptionInterceptor.instance())
-            .intercept(headersInterceptor)
-            .build();
 
-    logger.log(Level.INFO, String.format("%s initialized", session));
+    serverBuilder.addService(healthStatusManager.getHealthService());
+    serverBuilder.addService(new ActionCacheService(instance, config.getAcPolicy()));
+    serverBuilder.addService(new CapabilitiesService(instance));
+    serverBuilder.addService(
+        new ContentAddressableStorageService(
+            instance, /* deadlineAfter=*/ config.getCasWriteTimeout().getSeconds(), TimeUnit.SECONDS
+            /* requestLogLevel=*/ ));
+    serverBuilder.addService(
+        new ByteStreamService(
+            instance,
+            /* writeDeadlineAfter=*/ config.getBytestreamTimeout().getSeconds(),
+            TimeUnit.SECONDS));
+    serverBuilder.addService(
+        new ExecutionService(
+            instance,
+            config.getExecuteKeepaliveAfterSeconds(),
+            TimeUnit.SECONDS,
+            keepaliveScheduler,
+            getMetricsPublisher(config.getMetricsConfig())));
+    serverBuilder.addService(new OperationQueueService(instance));
+    serverBuilder.addService(new OperationsService(instance));
+    serverBuilder.addService(new AdminService(config.getAdminConfig(), instance));
+    serverBuilder.addService(new FetchService(instance));
+    serverBuilder.addService(ProtoReflectionService.newInstance());
+    serverBuilder.addService(new PublishBuildEventService(config.getBuildEventConfig()));
```

Comment:
Not a fan of this style change. It hides the actual modification here.

Reply:
+1, you can refactor later or stack diffs.

Reply:
reverted.  The diff is a little clearer now